invenio_deposit/ext.py

```diff
@@ -134,8 +134,7 @@ def init_app(self, app):
 
         @blueprint.record_once
         def extend_default_endpoint_prefixes(_):
-            """TODO."""
-            # FIXME remove hasattr() after 1.0.0a7 is released
```

Comment:
@lnielsen  @jirikuncar Thnx! Ok then I leave this PR as it is so as  to contain onyl the release.